Library/Homebrew/shims/linux/super/cc

```diff
@@ -311,6 +311,10 @@ class Cmd
     paths.map! { |path| prefix + path }
   end
 
+  def rpath_flags(prefix, paths)
+    paths.uniq.map { |path| prefix + path }
```

Comment:
Good question! A comment is justified here. The problem is that the `formula.lib` directory does not yet exist when the formula is being compiled and this function is called, so `formula.lib` gets pruned from the list of directories.